cli/src/lib.rs

```diff
@@ -726,6 +733,17 @@ fn build_cwd_verifiable(
         println!("Writing the .ts file");
         let ts_file = workspace_dir.join(format!("target/types/{}.ts", idl.name));
         fs::write(&ts_file, template::idl_ts(&idl)?)?;
+
+        // Copy out the TypeScript type.
+        if !&cfg.workspace.types.is_empty() {
```

Comment:
I wonder if we should copy or simply not write to `target/types/*.ts` when `types` is provided. I'm happy to copy for now though.